### ISSUE TITLE
Seriously?

### ISSUE THREAD

**Issue by KraXen72** [omnivore-app/omnivore#4459]
Seriously sold out to an AI startup?
...
I'm dissapointed.
Do better.

feel free to close this, i guess.

**Comment by gary-host-laptop** [comment]
I agree, and I also don't understand why would they have to delete all data and I need to export it? I guess after this it will be paywalled and you will be only able to have 5 articles before you need to cash in 20 bucks a month or some shait

**Comment by rlandolt** [comment]
And deleting the account also no longer works https://omnivore.app/settings/delete-my-account
It says it did it but it is still there...

**Comment by guissmo** [comment]
can this be self-hosted or are there any other alternatives?

**Comment by sotakal** [comment]
I’m so glad I went back to Instapaper a few months ago. It’s a private company but they seem to stick to their value proposition, and you know what you can expect from them.

The lack of development on Omnivore was a tell-tale sign that things were going nowhere.

**Comment by x9136** [comment]
Thank you, it was a useful app!

**Comment by felixbinder** [comment]
Time to fork it?

**Comment by jjafuller** [comment]
> can this be self-hosted or are there any other alternatives?

It can, but not really. The server component it tightly coupled to GCP and other third-party services. See https://github.com/omnivore-app/omnivore/issues/25 for the issues folks have been having over the years.

**Comment by ClariNerd617** [comment]
> Time to fork it? 

I'd be down, but based on what has been going down in #25 it may be a better idea to start fresh. Lots of tech debt.

**Comment by benedictjohannes** [comment]
I started to save news articles in this app for read later. I'm truly disappointed the way this is handled, by strong arming users under threat of account data deletion. I'm echoing this issue's title, SERIOUSLY???
The Android didn't even has TTS, while it *easily* tap in to the on-device TTS/accessibility service. I can't think of this app being open source as merely a gimmick to gain traction, after which the whole app/userbase would be sold for money. Sad.

**Comment by laril** [comment]
It seems this isn't easibly self-hostable, but could it make sense to host this somehow communally?  I'd be happy to pay for this.

**Comment by jrswab** [comment]
I too was disappointed by the announcement. It was the first read it later app that felt like it was done correctly and had everything I needed. Happy to see that I'm not the only one.

**Comment by samanthavbarron** [comment]
Disappointing is an understatement.

Unless someone forks it and addresses #25, I'll probably be migrating to FreshRSS and using NetNewsWire on iOS. Hopefully I'll be able to figure out something for TTS with that setup, that was a huge selling point for me on Omnivore.

RIP.

**Comment by Petemir** [comment]
> And deleting the account also no longer works https://omnivore.app/settings/delete-my-account It says it did it but it is still there...

Furthermore, the Export feature linked in the blog post nor e-mail seems to work. I am still waiting to receive my e-mail... this seems to be at odds with GDPR...

**Comment by ClariNerd617** [comment]
> > And deleting the account also no longer works https://omnivore.app/settings/delete-my-account It says it did it but it is still there...
> 
> Furthermore, the Export feature linked both in the blog post and e-mail does not seem to work either. I am still waiting to receive my e-mail... this seems to be at odds with GDPR...

Purely a hunch, but I'd say their servers are under heavy load due to a very large number of pissed-off users exporting their data all at once, as the hunt for alternatives commences. So far, I've been seeing some very interesting things about Wallabag.

**Comment by xplosionmind** [comment]
I got the red flag when I noticed that issues haven't been triaged or replied to in over one year!

**Comment by jeffgreco** [comment]
Unbelievably disappointed in this outcome, especially the short notice shutdown. Inexcusable.

**Comment by danclay** [comment]
> can this be self-hosted or are there any other alternatives?

I've been following a self-hosting oriented project hoarder-app/hoarder, but it is still very early in development and needs some polish, but seems promising with page archiving, full text search, cross-platform native clients, AI tagging, and more. For now I'm just going to primarily be using Obsidian web clipper. I've been trying to self host or use common file formats (e.g. MD for obsidian, do I really need some fancy proprietary notetaking system?) as much as possible recently since I've experienced this exact occurrence of a service suddenly shutting down with my data far too often-- suggest others try as well

**Comment by ron159** [comment]
I'm glad they made money through this software, but it's sad that a good software like this is gone.

**Comment by nicoska84** [comment]
> > And deleting the account also no longer works https://omnivore.app/settings/delete-my-account It says it did it but it is still there...
> 
> 
> 
> Furthermore, the Export feature linked both in the blog post and e-mail does not seem to work either. I am still waiting to receive my e-mail... this seems to be at odds with GDPR...

Still waiting too 😒

**Comment by xplosionmind** [comment]
There is also [*Readeck*](https://readeck.org)!

Still not super mature, but very promising 👀

**Comment by ragaskar** [comment]
my export completed successfully (although I am a relatively recent user and have probably fewer than a couple hundred items). 

Obviously, I wish the service was going to still be offered and/or was self-hostable -- omnivore seemed to be the best option I found in the survey I did of the various bookmarking sites out there, but I appreciate that an easy json-format export was made available. Thanks for the software, best of luck with the new organization.

**Comment by ilkka** [comment]
It was a good run! I'm glad we got to have this.

**Comment by devxan** [comment]
Best alternatives are probably [Linkwarden](https://github.com/linkwarden/linkwarden) (can archive pages, and can sync with browser bookmarks using [Floccus](https://github.com/floccusaddon/floccus/)! no RSS/newsletters afaik) or another bookmarking solution, like those in the previously mentioned repository [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted#bookmarks-and-link-sharing) (with [linkding](https://github.com/sissbruecker/linkding)+[injector](https://github.com/fivefold/linkding-injector) and [buku](https://github.com/jarun/Buku) really catching my eye)

**Comment by Petemir** [comment]
> > > And deleting the account also no longer works https://omnivore.app/settings/delete-my-account It says it did it but it is still there...
> > 
> > 
> > Furthermore, the Export feature linked both in the blog post and e-mail does not seem to work either. I am still waiting to receive my e-mail... this seems to be at odds with GDPR...
> 
> Purely a hunch, but I'd say their servers are under heavy load due to a very large number of pissed-off users exporting their data all at once, as the hunt for alternatives commences. So far, I've been seeing some very interesting things about Wallabag.

Yes, I thought the same, but if you give people 2 weeks to get their data out of your service, you better be sure they can actually do it. As @ragaskar , I also have less than a couple items (150, actually), yet I am still waiting.

> I'm glad they made money through this software, but it's sad that a good software like this is gone.

I'm also glad someone is getting money from OSS, but I'm not sure this was handled the best way.

**Comment by DDzwiedziu** [comment]
I'm here for the forks :fork_and_knife:

**Comment by codefruitio** [comment]
If people are looking for an interesting option for Text-to-Speech generation of their articles, check out my project at [blog2pod](https://github.com/blog2pod/)

**Comment by grapemix** [comment]
> > can this be self-hosted or are there any other alternatives?
> 
> It can, but not really. The server component it tightly coupled to GCP and other third-party services. See #25 for the issues folks have been having over the years.

I don't work for Omnivore. But based on the time and effort I try to self-hosted omnivore (https://github.com/omnivore-app/omnivore/issues/25#issuecomment-1974729755), it seems self-hosting omnivore is doable in a reasonable time. If you are in the industry, you will know new dev have to setup their dev environment, usually without any external third-party services.

It is fine to sell their company for money. But locking users and not providing a self-hosted solution to previously open-source project do bring damage to their reputation.

**Comment by kendofriendo** [comment]
this is literally the worst thing to ever happen to me in my entire life

**Comment by levifig** [comment]
> this is literally the worst thing to ever happen to me in my entire life

Pretty sure everyone on this thread will most certainly envy your life if this is the worst thing to ever happen to you... :)

**Comment by arcticpinecone** [comment]
I just wanted to state my total frustration with Omnivore owners. This bait and switch is fu***d up. If you want to move over to some other AI audio thing, fine.. but did you have to nuke this service to do so? Seriously just, wow.

**Comment by jeffgreco** [comment]
I am pretty sure I know the answer but any plans to inform users in the app?

**Comment by KraXen72** [comment]
if any1 decides to fork & maintain it please post it here.
although, self-hosting should become easier with #4465 from what i understand

**Comment by guissmo** [comment]
would love to see the steps or the README on self-hosting if / when it becomes available!

**Comment by RayBB** [comment]
For anyone that has looked at self-hosting, do you think it would be feasible to just fork the frontend and hook it up to the wallabag api? In theory it doesn't seem that hard but I'm not so familiar with nextjs.

**Comment by KarmaMole** [comment]
Did I literally lose all my saved links!? Is there any way to get them back?

**Comment by Perdellian** [comment]
> There is also [_Readeck_](https://readeck.org)!
> 
> Still not super mature, but very promising 👀

I only happened upon this thread because I was searching for whether Readeck supports TTS in its Android app before making the plunge towards setting it up, so I don't know all that much about Omnivore itself. Still, I'm hopping in with another potential alternative.

I've been using [Wallabag](https://wallabag.org/) for several years now, both self-hosted and paid via wallabag.it so I can (have my phone) read articles (to me) while I'm at work or otherwise not able to look at the screen. I had used Pocket before and found it overly cluttered, unreliable, and generally tetchy to use. Wallabag's Android app ain't the prettiest, but it gets the job done, and its TTS controls are excellent: seeking distance is configurable, and I can rewind with my watches' built-in media control apps or other bluetooth devices.

The only real reason I was looking into Readeck was so that I could perhaps have a better complementary/fallback option than Pocket (or manually using Firefox reader mode > select all > share to TTS-capable ereader) for sites that Wallabag's parser can't (yet?) handle.

**Comment by jeffgreco** [comment]
From a pure read-it-later perspective, I'm enjoying GoodLinks (
goodlinks.app )

**Comment by alain0** [comment]
> From a pure read-it-later perspective, I'm enjoying GoodLinks ( goodlinks.app )

And then wait until it gets acquired by an AI company 🤡

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.

**Comment by Podginator** [comment]
Reopening issues that still apply
